client/history/client.go

```diff
@@ -158,6 +158,7 @@ func (c *clientImpl) DescribeHistoryHost(
 		peer, err = c.peerResolver.FromHostAddress(request.GetHostAddress())
 	}
 	if err != nil {
+		c.logger.Error("peer could not be resolved for host.", tag.Error(err), tag.Address(request.GetHostAddress()))
```

Comment:
let's also include shardid or workflowid as a tag (whichever is used above)

Reply:
on it.